### ISSUE TITLE
Allow setting custom camera height, pitch and yaw

### ISSUE THREAD

**Issue by PavanLuca** [luanti-org/luanti#13725]
## Problem

I don't like that the third person visual has the crosshair behind the character's head, I'd really like to fight in third person but surely not with the crosshair there so it would be great to allow users to edit these 3 values, at least for the third person. These settings would also give a different and better fighting experience in certain servers, mostly where shooting.

## Solutions

Add camera height, pitch and yaw to settings.

## Alternatives

Add camera height, pitch and yaw to settings in a single settings entry, like for example (value) – (value) – (value).

**Comment by lhofhansl** [comment]
Agreed. The current limits on this are arbitrary, IMHO.

**Comment by PavanLuca** [comment]
Yes, sure, this is why I'd also suggest writing a clear warning under the options descriptions that tells the users that going over certain limits the camera view would be too weird to play clearly. Then, according to the situation and the server purpose, it's up to the user to set the most appropriate values to have a smooth experience. Below there are 2 examples of different camera positions for why I proposed this feature.

LEGO Bionicle Heroes view
![immagine](https://github.com/minetest/minetest/assets/123581658/76af15a1-7857-4313-906c-42bc6848e2a5)

Clash 3D games view
![immagine](https://github.com/minetest/minetest/assets/123581658/7f363f30-5f7a-4490-bac5-9789bf05f454)

**Comment by SmallJoker** [comment]
Included in the feature request #9798.
